README.md

```diff
@@ -44,5 +46,31 @@ Code Analysis for 8.x-1.16:
 ```
 
 ## Scoring
+Each module will be scored based on a number of objective criteria relating to:
+- Code Quality
+- Compliance
+- Customer Responsiveness
+- Ownership
+- Release Cadence
+- Security
 
+The scoring itself is tracked in a spreadsheet where each module has a tab to be filled out, and a report tab which is generated automatically. See this document:
+https://docs.google.com/spreadsheets/d/1iySyGQGs_G59O3hkBxRCX6fPXlVTnnXAyffCrbTVXqw/edit#gid=1023805792
```

Comment:
Not sure we should put a private doc in a public repo.

Reply:
fixed :)